contributors/guide/issue-triage.md

```diff
@@ -205,6 +205,11 @@ We aim for consistency across the entire project. However, if you notice an issu
 | **priority/backlog**  | General agreement that this is a nice-to-have, but no one's available to work on it anytime soon. Community contributions would be most welcome in the meantime, though it might take a while to get them reviewed if reviewers are fully occupied with higher-priority issues—for example, immediately before a release.| [**XXXX**]  | 
 | **priority/awaiting-more-evidence** | Possibly useful, but not yet enough support to actually get it done. | Mostly placeholders for potentially good ideas, so that they don't get completely forgotten, and can be referenced or deduped every time they come up | 
 
+### Define Milestones
+Consider in your triaging process also "/milestone" labeled issues. [Milestone maintainer](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md#milestone-maintainers) are responsible for setting this labels and interact with the [release team](https://github.com/kubernetes/sig-release/blob/master/release-team/README.md) to include enhancements into a release. 
+
+In case an issue has to be addressed within the next milestone, you ether include them through the regular release process (SIG Release Enhancement Team will reach out to the SIGs) or you have to request an [exception](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md) at the release team.
```

Comment:
"you ether include them" ==> "either include it"
"or you have to request" ==> "or request an exception **from** the release team"